prov/psm/src/psmx_init.c

```diff
@@ -558,6 +558,7 @@ static int psmx_getinfo(uint32_t version, const char *node, const char *service,
 	psmx_info->domain_attr->mr_iov_limit = 65535;
 	psmx_info->domain_attr->caps = PSMX_DOM_CAPS;
 	psmx_info->domain_attr->mode = 0;
+	psmx_info->domain_attr->mr_cnt = 10000;
```

Comment:
Is -1 analogous to 'unlimited' or something similar in the context of this new field?